packages/integration-tests/index.js

```diff
@@ -776,7 +773,7 @@ async function getExecutions(workflowName, stackName, bucket, maxExecutionResult
     stateMachineArn: kinesisTriggerTestStpFnArn,
     maxResults: maxExecutionResults
   });
-  return (orderBy(data.executions, 'startDate', 'desc'));
+  return (orderBy(data.executions, ['startDate'], ['desc']));
```

Comment:
I'm assuming that this is the case, but is changing these arguments to be two arrays instead of two strings functionally equivalent?

Reply:
Yes, this is functionally equivalent. The `orderBy` function will convert non-array args to arrays, so this change is not functionally necessary. I made the change to eliminate warnings that appear in the IDE because `orderBy`'s JSDoc defines the arguments as arrays, even though the function will properly deal with non-array values.